src/sqlfluff/rules/L003.py

```diff
@@ -578,35 +578,41 @@ def _eval(self, segment, raw_stack, memory, **kwargs):
             # we can assume that we're ok.
             self.logger.debug("    Indent deemed ok comparing to #%s", k)
 
-            # Given that this line is ok, consider if the previous line is a comment.
-            # If it is, lint the indentation of that comment.
-            if this_line_no - 1 in memory["comment_lines"]:
-                # The previous line WAS as comment.
-                prev_line = res[this_line_no - 1]
-                if this_line["indent_size"] != prev_line["indent_size"]:
-                    # It's not aligned.
-                    # Find the anchor first.
-                    anchor = None
-                    for seg in prev_line["line_buffer"]:
-                        if seg.is_type("comment"):
-                            anchor = seg
-                            break
-                    # Make fixes.
-                    fixes = self._coerce_indent_to(
-                        desired_indent="".join(
-                            elem.raw for elem in this_line["indent_buffer"]
-                        ),
-                        current_indent_buffer=prev_line["indent_buffer"],
-                        current_anchor=anchor,
-                    )
+            # Given that this line is ok, consider if the preceding lines are
```

Comment:
This change is smaller than it looks due to indentation changes. It just changes a check for "previous comment line" to "all contiguous preceding comment lines"

Reply:
FYI, GitHub has an option to remove whitespace when reviewing files, which can make changes like this easier to review:

![image](https://user-images.githubusercontent.com/10931297/133034161-c6bf9978-7458-4d43-ad3e-7b9a9b3d0f26.png)

Reply:
👍